src/Runner.Common/JobServerQueue.cs

```diff
@@ -665,6 +768,42 @@ private async Task UploadFile(UploadFileInfo file)
                 }
             }
         }
+
+    private async Task UploadResultsSummaryFile(ResultsSummaryUploadFileInfo file)
+        {
+            bool uploadSucceed = false;
+            try
+            {
+                Trace.Info($"Starting to upload summary file to results service {file.Type}, {file.Name}, {file.Path}");
+                
+
+                if (String.Equals(file.Type, ChecksAttachmentType.ResultsStepSummary, StringComparison.OrdinalIgnoreCase))
```

Comment:
will there every be other types of files?

Reply:
Not planned yet.